projects/noloan/common/src/main/java/hasadna/noloan/common/FirebaseAuthentication.java

```diff
@@ -20,8 +21,12 @@ public static FirebaseAuthentication getInstance() {
     return instance;
   }
 
-  public void signinAnonymusly() {
+  public FirebaseAuthentication() {
     auth = FirebaseAuth.getInstance();
+    user = auth.getCurrentUser();
```

Comment:
So now you get `user = auth.getCurrentUser();` in the constructor. Before it was in `signinAnonymusly()`.
Please test that this doesn't introduce issues with the main app, or alternatively, undo the change, or, run the `user = auth.getCurrentUser();` also at the start of `signinAnonymusly()` with a TODO to remove it if it's not needed..

Also, `auth = FirebaseAuth.getInstance();` had the same change. So anyway this needs to be checked in the main app.

Reply:
Tested, Working fine as far as I can see